test/fixtures/everything_at_once/results.json

```diff
@@ -1,22 +1,22 @@
 {
+    "status": "fail",
```

Comment:
Shouldn't this error?

Reply:
The spec says that the top level should only be `error` if no tests were able to run, so I think this is correct.